tests/test_config.py

```diff
@@ -1918,6 +1918,19 @@ def _test_quick_search_with_rcs(self,
         with self.assertRaises(TritonModelAnalyzerException):
             self._evaluate_config(new_args, yaml_content, subcommand='profile')
 
+    def test_bls_submodels(self):
+        """
+        Test that BLS submodels can be specified
```

Comment:
so this just makes sure there is no assert? Should there be a test using yaml since that is legal? Should there be a check that the submodels are supplied correctly in the resulting config?

Reply:
Yes, makes sure that the new option is legal and understood. 
Maybe, but we haven't done this extensive testing in the past. 
I thought about adding this, but I don't think it's needed here, since we are just reusing the same config primitive string that's already being tested.

Reply:
I've added a YAML check as well.